tests/unit_tests/eigen/eigen_masks_container.cpp

```diff
@@ -1,9 +0,0 @@
-/** Detray library, part of the ACTS project (R&D line)
```

Comment:
Why is this test removed?

Reply:
I thought it is basically covered by the mask store unittest

Reply:
ok Didn't know that there is mask store unittest..